UWP/HockeyAppDemoUwp/cs/App.xaml.cs

```diff
@@ -128,5 +146,23 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
 
         // Add any application contructor code in here.
         partial void Construct();
+
+        private bool HockeyAppConfigured = false;
+        private void ConfigureHockeyApp()
+        {
+            if (HockeyAppConfigured)
```

Comment:
Can we add this check in the SDK? After all we always expect configure to happen once so why not enforcing it?